src/tezos/operation.mli

```diff
@@ -2,14 +2,12 @@ open Crypto
 
 (* TODO: operation here means Manager_operation *)
 
-type parameters = {
-  entrypoint : string;
-  value : Michelson.t Data_encoding.lazy_t;
-}
 type transaction = {
```

Comment:
[OCaml newbie question] why is this type defined twice (in .ml and .mli)? I find this pretty error-prone (like modifying in .ml and not here)

Reply:
The mli is an interface for the ml(the implementation), so if you change one but not the other you're gonna get an error, think about mli in a similar fashion to a C header but checked against the implementation.